docs/visual-testing/integrations/csharp.md

```diff
@@ -441,6 +443,33 @@ await VisualClient.VisualCheck("C# full page config",
 
 <FullPageLimit />
 
+### Clip to an Element
+
+<ClippingDescription />
+
+Example:
+
+```csharp
+await VisualClient.VisualCheck("Visible Sale Banner",
+    new VisualCheckOptions()
+    {
+        ClipSelector = ".your-css-selector",
```

Comment:
Should we expose this in the docs if we want to have them migrate to ClipElement?

Reply:
I don't this so, we have no benefits of exposing this (ClipElement being more reliable)

Reply:
ok I've removed clipSelector from docs where clipElement is available. 

Please take a look at the updated [description](docs/visual-testing/_partials/_clipping-description.md) and let me know if the notes are applicable also to clipElement